bundle/manifests/webhook-service_v1_service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
I assume this file is auto-generated, but since I'm not too familiar with all of the certs stuff going on @sabinaaledort do you think this is a problem that it is suddenly generated?

Reply:
It was autogenerated.